### PR TITLE
Add docs for CSI unity support in CSM Operator

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Added docs for CSI Unity install support in csm-operator
Also, update unity driver version from 2.6 to 2.7 wherever applicable

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/756 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csmoperator/drivers/unity.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+kubectl get csm --all-namespaces
+```
+
+
+### Prerequisite
+
+1. Create namespace.
+   Execute `kubectl create namespace unity` to create the unity namespace (if not already present). Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'unity'.
+
+2. Create a file called `config.yaml` that has Unity XT array connection details with the following content
```
Comment (rensyct) [inline content/docs/deployment/csmoperator/drivers/unity.md:29]: Should this not be secret.yaml?

File: content/docs/deployment/csmoperator/release/_index.md [hunk 2]
```diff
@@ -6,14 +6,13 @@ Description: >
...
-- [Added support for CSI PowerStore Driver](https://github.com/dell/csm/issues/739)
+- [Added support for CSI Unity XT Driver](https://github.com/dell/csm/issues/756)
 
 
 ### Fixed Issues
-- [Unable to delete application pod when CSI-PowerStore is installed using CSM-Operator](https://github.com/dell/csm/issues/785)
+
 
 ### Known Issues
-CSM object does not track available deployment count when down scaling to n-1 , where n is number of nodes.
```
Comment (rensyct) [inline content/docs/deployment/csmoperator/release/_index.md:19]: Should not the Known issue come in Fixed Issues list

File: content/docs/deployment/csmoperator/release/_index.md [hunk 3]
```diff
@@ -6,14 +6,13 @@ Description: >
   Release notes for Dell Container Storage Modules Operator
 ---
 
-## Release Notes - Container Storage Modules Operator v1.2.0
+## Release Notes - Container Storage Modules Operator v1.3.0
```
Comment (rensyct) [inline content/docs/deployment/csmoperator/release/_index.md:9]: Upcoming release is 1.2.0.

**Review by rensyct — CHANGES_REQUESTED** [review]

File: content/docs/deployment/csmoperator/drivers/unity.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+| Parameter | Description | Required | Default |
+| --------- | ----------- | -------- |-------- |
+| replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, the excess pods will be in pending state until new nodes are available for scheduling. Default is 2 which allows for Controller high availability. | Yes | 2 |
+| namespace | Specifies namespace where the driver will be installed | Yes | "unity" |
+| fsGroupPolicy | Defines which FS Group policy mode to be used. Supported modes `None, File and ReadWriteOnceWithFSType` | No |"ReadWriteOnceWithFSType"|
+| ***Common parameters for node and controller*** |
+| X_CSI_UNITY_NODE_NAME_PREFIX | Prefix to add to each node registered by the CSI driver | Yes | "csi-node" 
+| X_CSI_UNITY_ALLOW_MULTI_POD_ACCESS | To enable sharing of volumes across multiple pods within the same node in RWO access mode | No | false |
+| X_CSI_UNITY_SYNC_NODEINFO_INTERVAL | Time interval to add node info to array. Default 15 minutes. Minimum value should be 1 | No | 15 |
+| CSI_LOG_LEVEL | Sets the logging level of the driver | true | info |
```
Comment (rensyct) [inline content/docs/deployment/csmoperator/drivers/unity.md:68]: Default value is debug

File: content/docs/deployment/csmoperator/release/_index.md (hunk 2 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/release/_index.md:19]: Done

File: content/docs/deployment/csmoperator/drivers/unity.md (hunk 1 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/unity.md]: Done

File: content/docs/deployment/csmoperator/release/_index.md (hunk 3 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/release/_index.md]: Done

File: content/docs/deployment/csmoperator/drivers/unity.md (hunk 4 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/unity.md:68]: For unity the default value is info as per the driver documentation

Comment (rensyct) [reply content/docs/deployment/csmoperator/drivers/unity.md:68]: But in csm-operator code, i see it set to debug for all drivers

**Review by rensyct — APPROVED** [review]

**Review by karthikk92 — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Pull request merged**